### PR TITLE
Hn updates

### PR DESCRIPTION
**Pull request by hknahal**

### CODE REVIEW

**Comment by hknahal** [comment]
@kcullion I updated the documentation with the suggestions you mentioned in https://github.com/icgc-argo/argo-docs/pull/285.

File: docs/submission/clinical-data-validation-rules.md [hunk 1]
```diff
@@ -19,29 +19,28 @@ The data dictionary contains certain data elements regarded as "identifiers". Th
...
 - Identifiers are assigned by each submitting program and must be unique within all the data submitted by that program (no duplicate IDs allowed).
-- These identifiers should be values that uniquely identify the entity they refer to.
-- Identifiers referring to the same entity should be consistent across different submissions from the submitting program, and should not be re-used to refer to two different entities. For example, the same donor should not be assigned different identifiers in different files or subsequent data submissions.
-- Identifiers cannot begin with DO, SP, SA, PD, TR or FU. These prefixes are reserved for ICGC-ARGO IDs.
+- Identifiers referring to the same entity should be consistent across separate program submissions and should not be re-used for different entities. For example, the same donor should not be assigned different identifiers in different files or subsequent data submissions.
+- Identifiers cannot begin with DO, SP, SA, PD, TR or FU. These prefixes are reserved for ICGC ARGO IDs.
```
Comment (rosibaj) [inline docs/submission/clinical-data-validation-rules.md:27]: for now (until we change anything, should we make a note about `case-sensitivity`?

Comment (hknahal) [reply docs/submission/clinical-data-validation-rules.md:27]: Good point @rosibaj - so right now, the IDs are still case-sensitive, right? I'll add a note about this.

Comment (rosibaj) [reply docs/submission/clinical-data-validation-rules.md:27]: yes right now they still are  - so at least a notice/warning for now until we change that feature.

File: docs/submission/faq.md [hunk 2]
```diff
@@ -10,19 +10,19 @@ platform_key: DOCS_FAQ
 
 1. How do I submit multiple values for a field? For example, I want to submit multiple symptoms for the `presenting_symptoms` field.
 
-   To include multiple values, separate values with a pipe delimiter '|' within your file. For example: `Anemia|Fatigue|Fever`.
+   When permitted, you can submit multiple values separated with a pipe delimiter '|'. For example: `Anemia|Fatigue|Fever`. Fields that allow multiple values will have these instructions specified in the notes section fo the data dictionary.
```
Comment (kcullion) [inline docs/submission/faq.md:13]: typo... change fo to `of`

**Pull request merged**